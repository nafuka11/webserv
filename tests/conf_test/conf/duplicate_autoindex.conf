server {
    listen 4242;

    autoindex on;
    autoindex off;
}