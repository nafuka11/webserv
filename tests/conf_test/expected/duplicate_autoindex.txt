webserv: duplicate "autoindex" directive in ./conf/duplicate_autoindex.conf:3
